presto-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3StorageClass.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive.s3;
+
+import com.amazonaws.services.s3.model.StorageClass;
```

Comment:
```suggestion
import com.amazonaws.services.s3.model.StorageClass.Standard;
import com.amazonaws.services.s3.model.StorageClass.IntelligentTiering;
```

Reply:
I think it's better not to static import this. Having `Standard` and `STANDARD` in the same scope is not helpful.